pkg/azurefile/nodeserver.go

```diff
@@ -206,7 +206,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		options := util.JoinMountOptions(mountFlags, []string{"loop"})
 		if strings.HasPrefix(fsType, "ext") {
 			// following mount options are only valid for ext2/ext3/ext4 file systems
-			options = util.JoinMountOptions(options, []string{"noatime", "barrier=0", "errors=remount-ro"})
+			options = util.JoinMountOptions(options, []string{"noatime", "barrier=1", "errors=remount-ro", "nostrictsync"})
```

Comment:
add `nostrictsync` in L161, and keep `errors=remount-ro` here

Reply:
done